pmg/api.py

```diff
@@ -465,10 +465,18 @@ def committee_meeting_attendance_summary():
     """
     Summary of MP attendance of committee meetings.
     """
+
+    # This is a temporary fix to only show attendance for members
+    # of the three major parties until we determine how to present
+    # faulty passed records for alternate members
+    parties = ['ANC', 'DA', 'EFF']
```

Comment:
Perhaps put this either into a constant at the top of the file, or on the MemberAttendance class (or whatever it is). Otherwise I suspect we'll change the one down the line, but not the other.